test/test_cpp20/ranges.cpp

```diff
@@ -45,4 +45,20 @@ TEST_CASE("ranges")
 
         REQUIRE((result == std::vector{ 2, 4, 6 }));
     }
+
+    // borrowable ranges. without enabling them, *result would fail to compile.
```

Comment:
"Without <feature> this test would not compile" is pretty generic. (Nit: the standard terminology is actually "borrowed range" as made concrete by the `borrowed_range` concept, rather than "borrowable range".)

Maybe just "Validate that rvalue winrt collections are borrowed ranges"?

Reply:
Borrow is getting pretty overloaded these days haha. Will also change this.